README.md

```diff
@@ -1,7 +1,95 @@
 # BestFlow
 We are going to fix bad chatbots!
 
-Hosted at: https://bestflow.netlify.app/
+Diagnostic tool for chatbot transcripts that help retailers identify poor or strong questions and intents through an easily digestible and intuitive display.
 
-This project uses:
-React, Axios, Node, Tailwind, MongoDB, Express and more :D
+We aim to:
+- Reduce time spent on chatbot optimization
+- Reduce bad chatbot intents
+- Display an average of 30% of diagnostic problems in any transcripts’ flow
+- Optimize the flow of at least three intents in x transcripts
+
+Each transcript will be displayed with a question and the responses it leads to.
+Each response also has a percentage displayed to show the chances of leading to a specific branch.
```

Comment:
I think proportion may be a better word than chance here unless we are really stressing brevity. Chance isn't really accurate to what is actually happening unless the user is interacting with the bot randomly right